src/lib/server/utils/crypto.ts

```diff
@@ -27,3 +27,23 @@ export const decrypt = (encryptedText: string, secretKey: string) => {
     decipher.final(),
   ]).toString()
 }
+
+export const encryptString = (message: string) => {
+  const secretKey = process.env.SECRET_KEY
+
+  if (!secretKey) {
+    throw new Error('Secret key not found')
+  }
+
+  return encrypt(message, secretKey)
+}
+
+export const decryptString = (message: string) => {
+  const key = process.env.SECRET_KEY
+
+  if (!key) {
```

Comment:
we should also call this `secretKey` for consistency?

Reply:
Sure yes!